### PR TITLE
fix sender email address

### LINKED ISSUE #1191: Sign in via email link - sender address misspelled
The sender is shown as open-**assistent**@laion.ai  - it should be assistant@

This could make it look like phishing or spam.

### PR DESCRIPTION
**Pull request by swaptr**
The sender email address has typo. This PR fixed that.
Fixes #1191

### CODE REVIEW

File: ansible/deploy-to-node.yaml [hunk 1]
```diff
@@ -177,7 +177,7 @@
             "{{ lookup('ansible.builtin.env', 'WEB_DISCORD_CLIENT_ID') }}"
           DISCORD_CLIENT_SECRET:
             "{{ lookup('ansible.builtin.env', 'WEB_DISCORD_CLIENT_SECRET') }}"
-          EMAIL_FROM: open-assistent@laion.ai
```
Comment (andreaskoepf) [inline ansible/deploy-to-node.yaml:180]: Sorry, we cannot easily change this.

**Review by andreaskoepf — CHANGES_REQUESTED** [review]

File: ansible/deploy-to-node.yaml (hunk 1 above)
Comment (ArgiesDario) [reply ansible/deploy-to-node.yaml:180]: @andreaskoepf could you provide some insight on how this should be fixed? cause I was thinking of fixing it the same way @swaptr did

**Review by andreaskoepf — APPROVED** [review]
good news, we now got the correct E-Mail address

File: ansible/deploy-to-node.yaml (hunk 1 above)
Comment (andreaskoepf) [reply ansible/deploy-to-node.yaml:210]: @ArgiesDario The problem was not the source change, but creation of the account/alias on the admin side.

**Pull request merged**